src/web/core/src/test/java/org/geoserver/web/admin/StatusPageTest.java

```diff
@@ -178,7 +178,7 @@ public void testModuleStatusPanelVersion() {
             index++;
         }
         if (found == -1) {
-            fail("Module jvm not found, required for version check");
+            fail("ServiceConformance jvm not found, required for version check");
```

Comment:
I believe this is not intended, probably result of search and replace.
The status page reports the presence of GeoServer own modules, core, extensions and community modules, which might have, or not, a relationship to conformance classes.